pkg/standalone/standalone.go

```diff
@@ -50,8 +51,64 @@ const (
 	DaprRedisContainerName = "dapr_redis"
 )
 
+func isInstallationRequired(installLocation, requestedVersion string) bool {
+	destDir := daprDefaultLinuxAndMacInstallPath
+	if installLocation == "" {
+		if runtime.GOOS == daprWindowsOS {
+			destDir = daprDefaultWindowsInstallPath
+		}
+	}
+
+	daprdBinaryPath := filepath.Join(destDir, daprRuntimeFilePrefix) //e.g. /usr/local/bin/daprd or c:\\daprd
+
+	//First time install?
+
+	_, err := os.Stat(daprdBinaryPath)
+	if os.IsNotExist(err) {
+		return true
+	}
+
+	var msg string
+
+	//what's the installed version?
+	v, err := utils.RunCmdAndWait(daprdBinaryPath, "--version")
+	if err != nil {
+		msg = fmt.Sprintf("unable to determine installed Dapr version at %s", destDir)
+		print.FailureStatusEvent(os.Stdout, msg)
+		return false
+	}
+	installedVersion := strings.TrimSpace(v)
+
+	//"latest" version requested. need to check the corresponding version
+	if requestedVersion == daprLatestVersion {
+		latestVersion, err := getLatestRelease(daprGitHubOrg, daprGitHubRepo)
+		if err != nil {
+			msg = fmt.Sprintf("latest Dapr version information could not be found - %s", err.Error())
+			print.FailureStatusEvent(os.Stdout, msg)
+			return false
+		}
+		latestVersion = latestVersion[1:]
+		if installedVersion == latestVersion {
+			requestedVersion = latestVersion
```

Comment:
changing requestedVersion here doesnt seem correct and complicates the logic,, setting a bool flag here will make code easy to follow

Reply:
yes. to make this simpler, it just returns `false` along with an `Info` message